src/improvements/tasks/eth/000-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -0,0 +1,876 @@
+
+  ----- DecodedStateDiff[39] -----
+    Who:               <a href="https://github.com/ethereum-optimism/superchain-registry/blob/c1bcf3601dfdf72f0fd4f5bade180b9c0f94d93b/superchain/configs/mainnet/soneium.toml#L57">0x88e529A6ccd302c948689Cd5156C83D4614FAE92</a>
+    Contract:          OptimismPortal2 - Soneium
+    Chain ID:          1868
+    Raw Slot:          0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc
+    Raw Old Value:     0x000000000000000000000000e2f826324b2faf99e513d16d266c3f80ae87832b
+    Raw New Value:     0x0000000000000000000000002d7e764a0d9919e16983a46595cfa81fc34fa7cd
+    Decoded Kind:      address
+    Decoded Old Value: 0xe2F826324b2faf99E513D16D266c3F80aE87832B
+    Decoded New Value: <a href="https://github.com/ethereum-optimism/superchain-registry/blob/b40cf4289c58e28eb1c791f9ad5724380b7516a7/validation/standard/standard-versions-mainnet.toml#L31">0x2D7e764a0D9919e16983a46595CfA81fc34fa7Cd</a>
+    Summary:           ERC-1967 implementation slot
+    Detail:            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.s in a proxy contract that follows the ERC-1967 standard.
+
+  ----- DecodedStateDiff[40] -----
+    Who:               <a href="https://github.com/ethereum-optimism/superchain-registry/blob/84bce73573f130008d84bae6e924163bab589a11/superchain/configs/mainnet/ink.toml#L55">0x88FF1e5b602916615391F55854588EFcBB7663f0</a>
+    Contract:          L1StandardBridge - Ink
+    Chain ID:          57073
+    Raw Slot:          0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc
+    Raw Old Value:     0x00000000000000000000000064b5a5ed26dcb17370ff4d33a8d503f0fbd06cff
+    Raw New Value:     0x00000000000000000000000078972e88ab8bbb517a36caea23b931bab58ad3c6
+    Decoded Kind:      address
+    Decoded Old Value: 0x64B5a5Ed26DCb17370Ff4d33a8D503f0fbD06CfF
+    Decoded New Value: <a href="https://github.com/ethereum-optimism/superchain-registry/blob/84bce73573f130008d84bae6e924163bab589a11/validation/standard/standard-versions-mainnet.toml#L38">0x78972E88Ab8BBB517a36cAea23b931BAB58AD3c6</a>
+    Summary:           ERC-1967 implementation slot
+    Detail:            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+
+  ----- DecodedStateDiff[41] -----
+    Who:               0x95703e0982140D16f8ebA6d158FccEde42f04a4C
+    Contract:          SuperchainConfig
+    Chain ID:          1
+    Raw Slot:          0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc
+    Raw Old Value:     0x00000000000000000000000053c165169401764778f780a69701385eb0ff19b7
+    Raw New Value:     0x0000000000000000000000004da82a327773965b8d4d85fa3db8249b387458e7
+    Decoded Kind:      address
+    Decoded Old Value: 0x53c165169401764778F780a69701385eb0FF19B7
```

Comment:
Link?

Reply:
This address is actually not in the registry anywhere, since it predates the standard-versions tracking. I linked to etherscan instead.